bolsa/captcha/services/TwoCaptchaResolverService.py

```diff
@@ -0,0 +1,19 @@
+from twocaptcha import TwoCaptcha
+from twocaptcha.api import ApiException
+from bolsa.captcha.CaptchaResolverServiceInterface import CaptchaResolverServiceInterface
+
+class TwoCaptchaResolverService(CaptchaResolverServiceInterface):
+
+    def __init__(self, credential):
+        self._credential = credential
+
+    async def resolve(self, site_key, url):
+        solver = TwoCaptcha(self._credential)
+
+        try:
+            solvedcaptcha = solver.recaptcha(site_key, url)
+        except ApiException as error:
+            raise error
+        except Exception as e:
+            raise ValueError('captcha não pode ser resolvido')
+        return solvedcaptcha['code']
```

Comment:
Curti a ideia de conseguir capturar uma exception específica da lib 👏 mas por outro lado, não estamos usando para fazer nenhum tratamento diferente, ou até mesmo log, acredito que podemos remover e usar algo mais genérico. 
Para tratar a exception, e os services sempre retornarem as mesmas exceptions, o que acha de usarmos custom exceptions? Faz sentido?
Ex:
```suggestion
        try:
            solvedcaptcha = solver.recaptcha(site_key, url)
        except Exception as e:
             raise CaptchaResolverException(f'Erro ao tentar resolver captcha: {e}')
        return solvedcaptcha['code']
```

Reply:
é melhor separar o CaptchaResolverException numa pasta de exceptions?

Reply:
Acho uma boa!